src/locales/fr.json

```diff
@@ -351,6 +351,16 @@
       "forceUploadData": "Forcer l'upload des données",
       "copyIdentifier": "Copier l'identifiant OpenPATH",
       "createdOn": "créé le"
+    },
+    "notifications": {
+      "quotaExpired": {
+        "title": "Enregistrement des déplacements désactivés",
+        "content": "Vous avez atteint la limite des {{maxDaysToCapture}} jours de trajets inclus dans votre offre."
+      },
+      "quotaAlmostExpired": {
+        "title": "Enregistrement des déplacements",
+        "content": "Vous avez presque atteint la limite des {{maxDaysToCapture}} jours de trajets inclus dans votre offre."
+      }
```

Comment:
It's out of scope, but do we plan to push a way to directly subscribe when the user receive this kind of notification? Otherwise I feel like this could be very sub-optimal for user transformation

Reply:
When you click on the notification, you arrive in cozy-settings/subscription page which : 
- show current offer
- show a button to change offer

But it could be improved.

Reply:
Neat, thanks